packages/data-stores/src/domain-suggestions/actions.ts

```diff
@@ -9,14 +9,30 @@ export const receiveCategories = ( categories: DomainCategory[] ) =>
 		categories,
 	} as const );
 
-export const receiveDomainSuggestions = (
+export const fetchDomainSuggestions = () =>
+	( {
+		type: 'FETCH_DOMAIN_SUGGESTIONS',
+	} as const );
+
+export const receiveDomainSuggestionsData = (
 	queryObject: DomainSuggestionQuery,
 	suggestions: DomainSuggestion[] | undefined
 ) =>
 	( {
-		type: 'RECEIVE_DOMAIN_SUGGESTIONS',
+		type: 'RECEIVE_DOMAIN_SUGGESTIONS_DATA',
```

Comment:
IMO, `DATA` is almost always a tautology, doesn't add meaning. Maybe make it `SUCCESS`.